example/src/client/bases/tables/furniture.ts

```diff
@@ -0,0 +1,104 @@
+import { TableApiHandler } from '../../../apis/tableApiHandler';
+import { BatchPayload, CheckSelect, SelectSubset } from '../../../types/common';
+import {
+  Furniture,
+  FurnitureCreateArgs,
+  FurnitureCreateManyArgs,
+  FurnitureDeleteArgs,
+  FurnitureDeleteManyArgs,
+  FurnitureFindFirstArgs,
+  FurnitureFindManyArgs,
+  FurnitureFindUniqueArgs,
+  FurnitureGetPayload,
+  FurnitureUpdateArgs,
+  FurnitureUpdateManyArgs,
+} from '../../../types/furniture';
+
+export class FurnitureDelegate {
+  private tableId = 'table_asdfasdfasdfasdf';
+  private tableApiHandler: TableApiHandler;
+
+  constructor(accessToken?: string) {
+    this.tableApiHandler = new TableApiHandler(accessToken);
+  }
+
+  async findUnique<T extends FurnitureFindUniqueArgs>(
+    args?: SelectSubset<T, FurnitureFindUniqueArgs>,
+  ): Promise<CheckSelect<T, Furniture, FurnitureGetPayload<T>>> {
+    return await this.tableApiHandler.findUniqueQueryApi(this.tableId, args);
+  }
+
+  async findUniqueOrThrow<T extends FurnitureFindUniqueArgs>(
+    args?: SelectSubset<T, FurnitureFindUniqueArgs>,
+  ): Promise<CheckSelect<T, Furniture, FurnitureGetPayload<T>>> {
+    const result = await this.tableApiHandler.findUniqueQueryApi<
+      SelectSubset<T, FurnitureFindUniqueArgs>,
+      CheckSelect<T, Furniture, FurnitureGetPayload<T>>
+    >(this.tableId, args);
+    if (result != null || result != {}) {
```

Comment:
result != {} will always be true
![image](https://user-images.githubusercontent.com/104710778/184104392-e0b51b34-76f2-46f9-a429-f2296199eaea.png)